### PR TITLE
Bagel loader Added

### PR DESCRIPTION
**Pull request by Asif1405**
Added loader for BageDB

BagelDB vector store and reader already available at Llama Index.

### CODE REVIEW

**Comment by Asif1405** [comment]
@logan-markewich
Hi.. 
Can you check out this PR too?? 

TIA

File: llama_hub/bagel/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+The Bagel Loader returns a set of texts corresponding to query embeddings or query texts retrieved from a BagelDB.
+The user initializes the loader with a BagelDB. They then pass in a query vector or a query text along with optional query parameters like metadata, where, where documents and include.
+
+## Usage
+
+Here's an example usage of the BagelReader.
+
+```python
+from llama_index import download_loader
```
Comment (jerryjliu) [inline llama_hub/bagel/README.md:11]: nit: canonical is now `from llama_hub.bagel.base import BagelReader` (no need for download_loader)

**Review by jerryjliu — COMMENTED** [review]
nice! one comment. also can you remove poetry.lock from being checked in?

File: llama_hub/bagel/README.md (hunk 1 above)
Comment (Asif1405) [reply llama_hub/bagel/README.md]: Changed in ReadMe..

Thanks

**Comment by Asif1405** [comment]
> nice! one comment. also can you remove poetry.lock from being checked in?

Done..

@jerryjliu

**Review by jerryjliu — APPROVED** [review]

**Comment by Asif1405** [comment]
Hi, @jerryjliu 

Can you point out the check fail??

**Comment by Asif1405** [comment]
@logan-markewich can you please look into it??

**Pull request merged**